gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/factories/GafferFactory.java

```diff
@@ -101,6 +104,32 @@ private static GafferSpec overrideGafferSpecConfig(final GafferSpec config, fina
         return opAuthoriser;
     }
 
+    private static ArrayList<Object> createOperationDeclaration(final GafferSpec config) {
+        ArrayList<Object> operations = new ArrayList<>();
+        if (config.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY) != null) {
+            operations.add(config.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY));
+        }
+        HashMap<String, Object> contents = new HashMap<>();
+        HashMap<String, String> contentsOfHandler = new HashMap<>();
+
+        contentsOfHandler.put("class", "uk.gov.gchq.gaffer.proxystore.operation.handler.GetProxyUrlHandler");
+
+        contents.put("operation", "uk.gov.gchq.gaffer.proxystore.operation.GetProxyUrl");
+        contents.put("handler", contentsOfHandler);
+
+        ArrayList<Object> objects = new ArrayList<>();
+        objects.add(contents);
+        if (!operations.isEmpty()) {
```

Comment:
You don't need to check if it is empty before a for each loop

Reply:
Updated